src/sagemaker/clarify.py

```diff
@@ -796,6 +969,9 @@ def __init__(
             endpoint_name (str): Sets the endpoint_name when re-uses an existing endpoint.
                 Cannot be set when ``model_name``, ``instance_count``,
                 and ``instance_type`` set
+            time_series_model_config (TimeSeriesModelConfig): Optional. A config object for
```

Comment:
Thought:

IMO, `ModelPredictedLabelConfig` is a tabular use-case equivalent of `TimeSeriesModelConfig`.

For that reason, it is strange different approaches in how we pass the values for tabular and timeseries use-case.

For tabular use-case, we pass it to run_explainability method, while for Timeseries, we, for a reason, use `ModelConfig`

Reply:
I understand your point that it's inconsistent. However, I think ultimately this is the better approach. Keeping `TimeSeriesModelConfig` as a parameter of `ModelConfig` means I don't have to touch `run_explainability`. Other members of the team also backed this approach over keeping `TimeSeriesModelConfig` as a standalone object as they don't consider them equivalents.